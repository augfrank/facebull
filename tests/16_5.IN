M0   C0   C1  2162
M1   C0   C2  2321
M2   C0   C3  2024
M3   C0   C4  2736
M4   C0   C5  2697
M5   C0   C6  2837
M6   C0   C7  2013
M7   C0   C8  2686
M8   C0   C9  2079
M9   C0   C11  2790
M10   C0   C13  2604
M11   C0   C14  2076
M12   C0   C15  2958
M13   C1   C0  2202
M14   C1   C4  2988
M15   C1   C6  2790
M16   C1   C7  2216
M17   C1   C8  2818
M18   C1   C10  2952
M19   C1   C13  2504
M20   C1   C14  2738
M21   C2   C3  2221
M22   C2   C5  2188
M23   C2   C6  2719
M24   C2   C8  2008
M25   C2   C11  2199
M26   C2   C14  2584
M27   C3   C1  2290
M28   C3   C4  2904
M29   C3   C5  2957
M30   C3   C7  2887
M31   C3   C8  2350
M32   C3   C10  2726
M33   C3   C12  2947
M34   C3   C13  2274
M35   C3   C15  2668
M36   C4   C6  2531
M37   C4   C7  2428
M38   C4   C8  2447
M39   C4   C9  2178
M40   C4   C10  2737
M41   C4   C11  2396
M42   C4   C12  2529
M43   C4   C14  2864
M44   C4   C15  2768
M45   C5   C0  2101
M46   C5   C1  2366
M47   C5   C2  2525
M48   C5   C4  2001
M49   C5   C8  2997
M50   C5   C10  2059
M51   C5   C12  2094
M52   C5   C14  2341
M53   C6   C1  2682
M54   C6   C3  2831
M55   C6   C4  2616
M56   C6   C5  2228
M57   C6   C7  2299
M58   C6   C8  2172
M59   C6   C9  2617
M60   C6   C10  2851
M61   C6   C12  2653
M62   C6   C13  2428
M63   C6   C14  2935
M64   C7   C0  2883
M65   C7   C1  2433
M66   C7   C2  2249
M67   C7   C3  2333
M68   C7   C5  2362
M69   C7   C8  2571
M70   C7   C10  2690
M71   C7   C11  2553
M72   C7   C12  2180
M73   C7   C14  2235
M74   C7   C15  2542
M75   C8   C0  2089
M76   C8   C2  2982
M77   C8   C3  2543
M78   C8   C5  2932
M79   C8   C6  2376
M80   C8   C9  2032
M81   C8   C12  2484
M82   C8   C13  2324
M83   C8   C14  2600
M84   C8   C15  2482
M85   C9   C0  2255
M86   C9   C1  2729
M87   C9   C2  2817
M88   C9   C3  2565
M89   C9   C4  2218
M90   C9   C5  2291
M91   C9   C8  2499
M92   C9   C10  2341
M93   C9   C14  2968
M94   C10   C6  2947
M95   C10   C7  2791
M96   C10   C8  2510
M97   C10   C14  2227
M98   C10   C15  2094
M99   C11   C4  2856
M100   C11   C6  2223
M101   C11   C7  2560
M102   C11   C8  2091
M103   C11   C10  2907
M104   C11   C13  2507
M105   C11   C14  2296
M106   C12   C4  2379
M107   C12   C5  2958
M108   C12   C8  2534
M109   C12   C9  2236
M110   C12   C10  2171
M111   C12   C13  2428
M112   C12   C14  2927
M113   C13   C0  2850
M114   C13   C1  2579
M115   C13   C2  2239
M116   C13   C3  2267
M117   C13   C4  2471
M118   C13   C9  2908
M119   C13   C10  2197
M120   C13   C12  2793
M121   C13   C14  2608
M122   C14   C0  2276
M123   C14   C1  2302
M124   C14   C5  2399
M125   C14   C8  2826
M126   C14   C10  2931
M127   C14   C11  2411
M128   C14   C12  2853
M129   C15   C1  2489
M130   C15   C2  2482
M131   C15   C4  2478
M132   C15   C5  2913
M133   C15   C6  2816
M134   C15   C7  2625
M135   C15   C9  2963
M136   C15   C10  2745
M137   C15   C11  2412
M138   C15   C13  2443
M139   C15   C14  2441
