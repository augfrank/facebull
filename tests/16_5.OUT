35177
2 13 24 34 41 48 57 72 80 90 98 100 110 115 123 139
